source/app/service-providers/windows/index.ts

```diff
@@ -256,6 +256,7 @@ export default class WindowProvider extends ProviderContract {
       await this._stateContainer.init(Object.fromEntries(this._windowState))
     }
     const tmpObject = await this._stateContainer.get()
+    if (tmpObject === null) this._logger.error('[Window Manager] tmpObject is null');
```

Comment:
This check should be unnecessary after I've merged the PR by @UFOMelkor, since there is no way right now that `get()` will return null.

Reply:
Also, I just saw that this change is also being obliterated by your other change to the `get()` accessor below.